data_summaries/Open-Platypus.json

```diff
@@ -0,0 +1,556 @@
+        "Paper Title": "",
+        "Dataset URL": "https://huggingface.co/datasets/jondurbin/airoboros-gpt4-1.4.1",
+        "GitHub URL": "https://github.com/jondurbin/airoboros",
+        "Hugging Face URL": "https://huggingface.co/datasets/jondurbin/airoboros-gpt4-1.4.1",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "Open-Platypus",
+        "Collection URL": "https://platypus-llm.github.io",
+        "Languages": [
+            "Code",
+            "English"
+        ],
+        "Task Categories": [
+            "Brainstorming",
+            "Code",
+            "Creative Writing",
+            "Question Answering",
+            "Question Understanding"
+        ],
+        "Text Sources": [
+            "OpenAI ChatGPT"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-4",
+            "OpenAI ChatGPT"
+        ],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [],
+        "Creators": [],
+        "Licenses": [
+            {
+                "License": "CC BY-NC 4.0",
+                "License URL": "https://huggingface.co/datasets/jondurbin/airoboros-gpt4-1.4.1"
+            },
+            {
+                "License": "OpenAI",
+                "License URL": "https://github.com/jondurbin/airoboros"
+            }
+        ],
+        "License Notes": "Uses OpenAI to generate synthetic data.",
+        "License Verified By": "Ariel Lee",
+        "Dataset Filter IDs": [
+            "airoboros"
+        ]
+    },
+    "op-arb": {
+        "Unique Dataset Identifier": "op-arb",
+        "Dataset Name": "arb",
+        "Paper Title": "ARB: Advanced Reasoning Benchmark for Large Language Models",
+        "Dataset URL": "https://arb.duckai.org",
+        "GitHub URL": "https://github.com/TheDuckAI/arb",
+        "Hugging Face URL": "",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2307.13692",
+        "Semantic Scholar Corpus ID": 260155126,
+        "Collection": "Open-Platypus",
+        "Collection URL": "https://platypus-llm.github.io",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Argumentative Reasoning",
+            "Commonsense Reasoning",
+            "Chain-of-Thought",
+            "Logical and Mathematical Reasoning",
+            "Question Answering"
+
+        ],
+        "Text Sources": [
+            "bar exam",
+            "berkeley problems in mathematics",
+            "major american universities ph.d. qualifying questions and solutions",
+            "mcat tests",
+            "problems and solutions on electromagnetism",
+            "problems and solutions on mechanics",
+            "problems and solutions on optics",
+            "problems and solutions on thermodynamics and statistical mechanics",
```

Comment:
Are these all from science textbooks or something? Wondering if we can consolidate to name the source as precisely as we can

Reply:
consolidated these since they're all from world scientific